pullrequests/PollPR.py

```diff
@@ -251,24 +282,53 @@ def PRBuilding(data, ErroCounter, g, org, counter, myfile, tbd_list, timetoken):
         ErroCounter = ErroCounter+1
 
     USER = data["user"]["login"]
-    user = g.get_user(USER)
+    UserOK=False
+    #global organization_users
 
-    if (org.has_in_members(user)):
-        print(
-            f"---> The user '{USER}' is a member of the organization '{ORGANIZATION}'.")
+    if (USER in organization_users):
+        UserOK=True
+        print(f"---> Earlier check found:'{USER}' is a member of the organization '{ORGANIZATION}'.")
```

Comment:
Should also store if earlier check found out that user is **not** member of the organization.

Reply:
I will create separate issue for this, in order to get current version of rate limiting out asap